CS3219-AY2223S1/cs3219-project-ay2223s1-g12
# Add simple sign up login page

Fixes #40 

Summary:
- Created a login page component with similar UI as sign up page
- Linked login page to route at `/login`
- Connect to backend login with axios
- Errors are displayed using a snackbar positioned on top center
- Successful login are redirected to `/home` page temporary placeholder

Todos [DONE]:
- Pending #34 and #23: Approval of unique username PR and login PR so that I can connect to the backend with axios
- Routing to home page after successful login

How to test:
1. `npm i`, `npm run dev` on user-service in one terminal
2. `npm i`, `npm start` on frontend directory in one terminal
3. Go to `http://localhost:3000/login` to reach the login page
4. Enter the details accordingly and click `Login` button
5. If error, you will see a snackbar pop up
![image](https://user-images.githubusercontent.com/65227564/191881102-9bf51fc9-162c-42ea-9df0-af2f25198fdd.png)

7. If successful, you will be redirected to `/home` page
![image](https://user-images.githubusercontent.com/65227564/191881136-805c59f5-3479-4389-bc80-b12eb2bdd92f.png)
